device.js

```diff
@@ -49,11 +50,20 @@ class LoupedeckDevice extends EventEmitter {
         else if (this.host) this.connection = new WSConnection({ host: this.host })
         // Autodiscover
         else {
-            for(const type of [SerialConnection, WSConnection]) {
-                const args = await type.discover()
-                if (!args) continue
-                this.connection = new type(args)
-                break
+            const devices = await listDevices()
```

Comment:
If `.list()` was a static method, this could just be `const devices = await this.constructor.list()`

Reply:
Implemented in [221d7ef](https://github.com/foxxyz/loupedeck/pull/12/commits/221d7ef9c8f3a716abceb59c300dbe3f160d8c5b)